CSharpMath.CoreTests/TypesetterTests.cs

```diff
@@ -33,15 +33,15 @@ public class TypesetterTests {
     void TestOuter(string latex, int rangeMax, double ascent, double descent, double width,
         params System.Action<IDisplay<TFont, TGlyph>>[] inspectors) =>
       TestList(rangeMax, ascent, descent, width, 0, 0, LinePosition.Regular, Range.UndefinedInt, inspectors)
-      (Typesetter.CreateLine(LaTeXParserTest.ParseLaTeX(latex), _font, _context, LineStyle.Display));
+      (ParseLaTeXToDisplay(latex));
 
-    [Theory, InlineData("x"), InlineData("2")]
-    public void TestSimpleVariable(string latex) =>
+    [Theory, InlineData("x", "𝑥"), InlineData("2", "2"), InlineData(".", "."), InlineData("𝑥", "𝑥")]
+    public void TestSingleCharacter(string latex, string text) =>
       TestOuter(latex, 1, 14, 4, 10,
         d => {
           var line = Assert.IsType<TextLineDisplay<TFont, TGlyph>>(d);
           Assert.Single(line.Atoms); // have to think about these; doesn't really work atm
```

Comment:
is this comment outdated?

Reply:
@verybadcat wrote it but I think `Atoms` work fine.